control-plane/stor-port/src/types/v0/store/volume.rs

```diff
@@ -212,6 +213,9 @@ pub struct VolumeSpec {
     /// Data encryption.
     #[serde(default)]
     pub encrypted: bool,
+    /// Pool's blobstore cluster size required for replicas of this volume.
+    #[serde(default = "default_pool_cluster_size")]
+    pub cluster_size: u32,
```

Comment:
Let's keep Option, allows us to identify if cluster_size has been explicitly specified or not?

```suggestion
    #[serde(default, skip_serializing_if = "Option::is_none")]
    pub cluster_size: Option<u32>,
```

Reply:
> Let's keep Option, allows us to identify if cluster_size has been explicitly specified or not?

No, let's keep it just u32 here. Will be a lot of change again otherwise. If we need to figure that out, it can be figured out from CR?

Reply:
Ok